### PR TITLE
rafthttp: simplify initialization funcs

### PR DESCRIPTION
**Pull request by xiang90**
@heyitsanthony @gyuho

### CODE REVIEW

File: rafthttp/peer.go [hunk 1]
```diff
@@ -117,14 +117,16 @@ type peer struct {
 	stopc  chan struct{}
 }
 
-func startPeer(transport *Transport, urls types.URLs, local, to, cid types.ID, r Raft, fs *stats.FollowerStats, errorc chan error) *peer {
-	plog.Infof("starting peer %s...", to)
-	defer plog.Infof("started peer %s", to)
+func startPeer(transport *Transport, urls types.URLs, id types.ID, fs *stats.FollowerStats) *peer {
```
Comment (heyitsanthony) [inline rafthttp/peer.go]: I think `to` is more descriptive than `id`

Comment (xiang90) [inline rafthttp/peer.go]: it is the id of this started peer. but from the pipeline perspective, it is the destination. i can add a to var and assign it to it.

Comment (heyitsanthony) [inline rafthttp/peer.go]: that's still confusing-- would local/remote be better?

Comment (xiang90) [inline rafthttp/peer.go]: will try.

**Comment by xiang90** [comment]
@heyitsanthony changed `to` to `peerID`. everything seems to be fit together.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**